estuary/models/productversion.py

```diff
@@ -0,0 +1,9 @@
+from neomodel import StringProperty
+
+from estuary.models.base import EstuaryStructuredNode
+
+
+class ProductVersion(EstuaryStructuredNode):
+    """Definition of RHEL Product Version in Neo4j."""
+
+    name = StringProperty()
```

Comment:
```suggestion
    name = StringProperty(unique=True, index=True)
```

Reply:
According to neomodel documentation, unique_index argument is the same as using unique and index arguments. WDYT?

Reply:
Sounds good!